core/state_processor.go

```diff
@@ -747,9 +770,11 @@ func (p *StateProcessor) execInParallelSlot(slotIndex int, txReq *ParallelTxRequ
 			// can skip current slot now, since slotDB is always after current slot's merged DB
 			// ** idle: all previous Txs are merged, it will create a new SlotDB
 			// ** queued: it will request updateSlotDB, dispatcher will create or reuse a SlotDB after previous Tx results are merged
-			if index == slotIndex {
-				continue
-			}
+
+			// with copy-on-write, can not skip current slot
```

Comment:
can we delete this

Reply:
it was disabled when both ReUse & Copy-On-Write are enabled, since Reuse is disabled, I can reopen this commented code.